src/ingestendpoint/secret_test.go

```diff
@@ -11,129 +11,131 @@ import (
 	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	testPaasToken              = "test-paas-token"
-	testAPIToken               = "test-api-token"
-	testDataIngestToken        = "test-data-ingest-token"
-	testUpdatedDataIngestToken = "updated-test-data-ingest-token"
+	paasToken              = "test-paas-token"
```

Comment:
why remove the `test` prefix?

Reply:
Why keep it? This is test file by definition, no need to keep boilerplate in consts naming.

Reply:
they can bleed into the actual `secret.go`, which is not cool

Reply:
can lead to confusing during testing. it's just easier to keep the prefix, also makes it clear it's a value only used for tests and not defined outside those.

Reply:
I didn't know that 😮 
(I still like the prefix though 😅 )

Reply:
Democracy won ;).